src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -15,7 +15,7 @@ import {
   updateSortByPropertyNameThunk,
   updatePaginationAction,
 } from '../actions';
-import { getFindFormsAppState, mvpEnhancements } from '../helpers/selectors';
+import { getFindFormsAppState } from '../helpers/selectors';
```

Comment:
Are we at a place where we can also remove the code for the `mvpEnhancements` selector itself and the Flipper feature?

Reply:
That's a good point. I think it's safe. The next improvement for FaF might be weighted results and 508's. 

And even if we do come up with another improvement visibly that needs to be behind the feature flag, I can just add the selector back in verses forgetting about it and it becoming tech debt.